styles/tokens/masthead.json

```diff
@@ -317,13 +308,166 @@
           "themable": true
         },
         "font-weight": {
-          "comment": "The font weight of the selector in the Masthead with Extended Navigation.",
+          "comment": "The font weight of the selector, found in a Masthead with Extended Navigation.",
           "value": "300",
           "file": "settings",
           "themable": true
         },
+        "dropdown-background-color": {
+          "comment": "The background color of the Selector Dropdown, found in a Masthead with Extended Navigation..",
```

Comment:
There is an extra period here.

Reply:
![tenor-42686865](https://user-images.githubusercontent.com/4342363/111218672-68432780-85ad-11eb-8d1c-c958dd3c226f.gif)
ok I fixed it!